types/queries/morpheme_tags.sql

```diff
@@ -4,7 +4,8 @@ select
   abbreviation_system.short_name as system_name,
   morpheme_tag.gloss,
   morpheme_tag.title,
-  abstract_morpheme_tag.description,
+  morpheme_tag.description,
+  morpheme_tag.segment_type as "segment_type: SegmentType",
```

Comment:
So this is required for sqlx to interpret the `segment_type` postgres enum as the rust type `SegmentType` otherwise it won't do that mapping on its own afaik.